### DISCUSSION TITLE
vimformedits?

### DISCUSSION THREAD

**Discussion by pbwn** [Jelmerro/Vieb#525] [General]
i was writing in a form field in vieb and pressed some unknown combination of keys and gvim opened with the text i had typed in the form field. The file created was in a dir called vimformedits and i later found it in ~/.config/Vieb/. I can't find anything about this in :help ie how to trigger it. It would be fantastic if i can move to vim to type a comment and then jump back to vieb (i map `:inoremap <CR> <CR><toNormalMode>` which makes writing on a forum in vieb a bit difficult), but how to do it on purpose? Thanks

**Comment by Jelmerro** [comment]
The feature is called editWithVim, you can find it using `:help editWithVim`. It by default triggers with `<C-i>`, so control then `i` while inside insert mode. You can customize the editor using the `vimcommand` setting. Let me know if this is what you meant.
